packages/front-end/components/Settings/Team/InviteModal.tsx

```diff
@@ -81,6 +85,19 @@ const InviteModal: FC<{ mutate: () => void; close: () => void }> = ({
     );
   }
 
+  // Hit a hard cap and needs to contact sales to increase the number of seats on their license
+  if (showContactSupport) {
+    return (
+      <Modal open={true} close={close} size="md" header={"Reached seat limit"}>
+        <div className="my-3">
+          Whoops! You reached the seat limit on your license. To increase your
+          number of seats, please contact{" "}
+          <a href="mailto:sales@growthbook.io">sales@growthbook.io</a>.
```

Comment:
Small suggestion - set this to open in a new tab.

```suggestion
          <a href="mailto:sales@growthbook.io" target="_blank" rel="noreferrer">
            sales@growthbook.io
          </a>.
```

Reply:
Good idea